android/src/main/java/com/thebluealliance/androidclient/activities/HomeActivity.java

```diff
@@ -55,7 +55,8 @@
 
     private static final String MAIN_FRAGMENT_TAG = "mainFragment";
 
-    @Inject TBAStatusController mStatusController;
+    @Inject
+    TBAStatusController mStatusController;
```

Comment:
Moreover, Reformat Code actually sticks this back on a different line, even after applying the proper code style.